feature/home/src/main/java/com/epmedu/animeal/home/presentation/HomeScreen.kt

```diff
@@ -44,7 +44,8 @@ fun HomeScreen() {
         HomeScreenUI(
             state = state,
             bottomSheetState = bottomSheetState,
-            onScreenEvent = viewModel::handleEvents
+            onScreenEvent = viewModel::handleEvents,
+            onWillFeedEvent = viewModel::handleWillFeedEvent
```

Comment:
I think that there should be a way to use 1 method for both

Reply:
To achieve this, we can either make both events extend a common interface (which may result in high coupling), or create a GlobalEvents interface that all UI events can extend.

Although keeping events separated may seem like boilerplate code, it provides greater control over the events handled by the screen and allows only necessary events to be passed. Additionally, this approach enables handlers to manage each feature, keeping view models cleaner.

Reply:
Ok, let's keep them separated then